src/olympia/addons/views.py

```diff
@@ -1127,17 +1125,16 @@ def get_language_packs_queryset_with_appversions(self, application, appversions)
             .distinct()
         )
 
-    @method_decorator(cache_page(60 * 60 * 24))
-    def dispatch(self, *args, **kwargs):
-        return super().dispatch(*args, **kwargs)
-
     def list(self, request, *args, **kwargs):
         # Ignore pagination (return everything) but do wrap the data in a
         # 'results' property to mimic what the default implementation of list()
         # does in DRF.
         queryset = self.filter_queryset(self.get_queryset())
         serializer = self.get_serializer(queryset, many=True)
-        return Response({'results': serializer.data})
+        response = Response({'results': serializer.data})
+        # Cache on the CDN/clients for 24 hours.
+        patch_cache_control(response, max_age=60 * 60 * 24)
+        return response
```

Comment:
kind of a nit, but it's irritating (to me) that we always have to do this in 3 separate statements - if only `patch_cache_control` returned the `response` it could be written as a single statement 🤷

Reply:
Yeah... the better path would be to use the `cache_control` decorator, but any time that header needs a dynamic value or be added or not depending on something else, it doesn't work... I'm not a big fan of `patch_cache_control` either but also not really a fan or writing our own helper just for 3 lines of code.